.github/workflows/repo.hex.pm.yml

```diff
@@ -61,10 +75,37 @@ jobs:
                   aws s3 cp "${tarball}" "s3://${{ env.AWS_S3_BUCKET }}/docs/${tarball}" \
                       --cache-control "public,max-age=3600" \
                       --metadata "{\"surrogate-key\":\"${surrogate_key}\",\"surrogate-control\":\"public,max-age=604800\"}"
-                  purge_key "${surrogate_key}"
-                  sleep 2
-                  purge_key "${surrogate_key}"
-                  sleep 2
-                  purge_key "${surrogate_key}"
+                  purge "${surrogate_key}"
               fi
           done
+      - name: Upload Precompiled to S3
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.HEX_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.HEX_AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: ${{ secrets.HEX_AWS_REGION }}
+          AWS_S3_BUCKET: ${{ secrets.HEX_AWS_S3_BUCKET }}
+        run: |
+          mv elixir-otp-${{ matrix.otp }}.zip elixir.zip
```

Comment:
I don't think we need this `mv` right? We can directly `aws s3 cp` the first one?

Reply:
it might not be necessary but as I can not list the items in the bucket I limited myself to copy and paste from [bob](https://github.com/hexpm/bob/blob/main/priv/scripts/elixir/elixir.sh#L81).

Reply:
https://github.com/elixir-lang/elixir/pull/12242/commits/6e75af90a9384b75f76f3dbd0c0650b9f7ffe97b